packages/dai/test/eth/MulticallService.spec.js

```diff
@@ -1,18 +1,28 @@
 import TestAccountProvider from '@makerdao/test-helpers/src/TestAccountProvider';
 import { buildTestMulticallService } from '../helpers/serviceBuilders';
+import schemas from '../helpers/schemas';
+import addresses from '../../../dai-plugin-mcd/contracts/addresses/testnet.json';
+import { first } from 'rxjs/operators';
+// import { promiseWait } from '../../src/utils';
 
 let service;
 let watcher;
 let address;
+let addressWithProxy;
 
-beforeAll(async () => {
+beforeEach(async () => {
   service = buildTestMulticallService();
   await service.manager().authenticate();
+  addressWithProxy = service.get('web3').currentAddress();
   address = TestAccountProvider.nextAddress();
   watcher = service.createWatcher({ interval: 'block' });
+  service.registerSchemas(schemas);
+  service._addresses = addresses;
```

Comment:
this seems hacky... but i guess it goes away once you move MCD-specific tests to the plugin?

Reply:
Yep.